### PR TITLE
Make screenshot plugin more reliable

### PR DESCRIPTION
**Pull request by ThatOtherPerson**
Adds concurrency limit and retries on failure to screenshot plugin

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit f739087561a4d5b84c2ad9aa4ecc7aeed5d7d2cd

https://app.netlify.com/sites/using-postcss-sass/deploys/5b5ba3ef67610c7b1d1688f1

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit f739087561a4d5b84c2ad9aa4ecc7aeed5d7d2cd

https://app.netlify.com/sites/gatsbyjs/deploys/5b5ba3ee67610c7b1d1688e9

File: packages/gatsby-transformer-screenshot/src/gatsby-node.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
 const crypto = require(`crypto`)
 const axios = require(`axios`)
+const Queue = require(`better-queue`)
 const { createRemoteFileNode } = require(`gatsby-source-filesystem`)
 
 const SCREENSHOT_ENDPOINT = `https://h7iqvn4842.execute-api.us-east-2.amazonaws.com/prod/screenshot`
+const LAMBDA_CONCURRENCY_LIMIT = 50
```
Comment (thebigredgeek) [inline packages/gatsby-transformer-screenshot/src/gatsby-node.js]: Should this be configurable via environment variable, perhaps?

Comment (KyleAMathews) [reply packages/gatsby-transformer-screenshot/src/gatsby-node.js]: There's only one lambda that we own so unless we make the endpoint configurable, there's no reason not to hard code this.

**Pull request merged**